### PR TITLE
Dependency cleanup and updates

### PR DESCRIPTION
**Pull request by harshithmohan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,95 +76,15 @@
...
-    "stylelint": "^14.16.1",
-    "stylelint-config-standard": "^29.0.0",
+    "stylelint": "^15.7.0",
+    "stylelint-config-standard": "^33.0.0",
     "stylelint-config-tailwindcss": "^0.0.7",
+    "stylelint-stylistic": "^0.4.2",
     "tailwindcss": "^3.3.2",
     "tailwindcss-text-fill-stroke": "^2.0.0-beta.1",
-    "typescript": "4.9.5",
+    "typescript": "5.0.4",
```
Comment (harshithmohan) [inline package.json:85]: `@typescript-eslint/eslint-plugin` doesn't support >=5.1.0

**Comment by harshithmohan** [comment]
It works perfectly without any of the resolutions 🤷‍♂️

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jmmserver-webui",
+  "name": "shoko-webui",
```
Comment (revam) [inline package.json:2]: Didn't know it was still jmm here 😅

**Comment by revam** [comment]
> It works perfectly without any of the resolutions 🤷‍♂️

What were the resolutions used for?

**Comment by hidden4003** [comment]
@revam security updates usually when package in question doesn't care to update dependencies

**Pull request merged**